Jenkinsfile

```diff
@@ -0,0 +1,137 @@
+stage 'Parallel Build'
+
+def variants = [
+    'clang.debug.unity',
+    'clang.debug.nounity',
+    'gcc.debug.unity',
+    'gcc.debug.nounity',
+    'gcc.release.unity',
+    'coverage'] as String[]
+
+// create a map of all builds
+// that we want to run. The map
+// is string keys and node{} object values
+def builds = [:]
+for (int index = 0; index < variants.size(); index++) {
+    def bldtype = variants[index]
+    builds[bldtype] = {
+        node('rippled-dev') {
```

Comment:
not in our case, in fact it would break things. The node "name" tells it what kind of node to run this job on - in our case, we can only run on slaves marked as `rippled-dev`. See docs:  https://jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#node-allocate-node.